api/serializers.py

```diff
@@ -62,6 +62,20 @@ def create(self, validated_data):
         return user
 
 
+class UserSerializerWithAssets(UserSerializer):
+    assets_allocated = serializers.SerializerMethodField()
+
+    def get_assets_allocated(self, obj):
+        assets = Asset.objects.filter(assigned_to__user=obj)
+        my_assets = [asset for asset in assets]
```

Comment:
A queryset is iterable (that's why the statement `for asset in assets` in the list comprehension above works), so there is no need to convert it to a list. The statement below is more succint.
```
AssetSerializer(assets, many=True)
```